pywbem/cim_obj.py

```diff
@@ -2853,6 +2855,148 @@ def tomof(self, indent=0, maxline=MAX_MOF_LINE):
 
         return u''.join(mof)
 
+    @staticmethod
+    def from_class(klass, namespace=None,
+                   property_values=None,
+                   include_missing_properties=True,
+                   include_path=True, strict=False,
+                   include_class_origin=False):
+        """
+        Create a new CIMInstance from the input CIMClass using the
+        property_values parameter to complete properties and the other
+        parameters to filter properties, validate the properties, and
+        optionally set the path component of the CIMInstance.
+        No CIMProperty qualifiers are included in the created instance and the
+        `class_origin` attribute is transfered from the class only if the
+        `include_class_origin` parameter is `True`.
+
+        Parameters:
+          klass (:class:`pywbem:CIMClass`)
+            CIMClass from which the instance will be constructed.  This class
+            must include qualifiers and should include properties from any
+            superclasses to be sure it includes all properties that are to be
+            built into the instance, in particular any key properties if the
+            `include+path` parameter is `True`. See
+            :meth:`~pywbem.CIMInstanceName.from_class` for further requirements
+            on the class.
+
+          namespace (:term:`string`):
+            Namespace to be included in the path component of the returned
+            CIMInstance if `include_path` parameter is `True`.
+
+          property_values (:class:`py:dict` or `NocaseDict`_):
+            Dictionary containing name/value pairs where the names are the
+            names of properties in the class and the properties are the
+            property values to be set into the instance properties. The values
+            must match the type defined for the property in the class. If a
+            property is in the property_values dictionary but not in the class
+            a ValueError exception is raised.
+
+          include_missing_properties (:class:`py:bool`):
+            Determines if properties not in the `property_values` parameter are
+            included in the instance.
+
+            If `True` all properties from the class are included in the new
+            instance including those not defined in `property_values` parameter
+            with any value from the class. This is the `Default`.
```

Comment:
You indicated you had improved the phrase "with any value from the class.", but it is still there. Given that you clarified that the class-defined default value or NULL is used, I suggest to replace that phrase with: "with the default value defined in the class if one is defined, or otherwise `None`".

Reply:
DONE. That is clearer.